frigate/detectors/plugins/deepstack.py

```diff
@@ -48,7 +48,6 @@ def detect_raw(self, tensor_input):
         with io.BytesIO() as output:
             image.save(output, format="JPEG")
             image_bytes = output.getvalue()
-        data = {"api_key": self.api_key}
```

Comment:
This removal looks somewhat suspicious. Should probably be passed to the POST request  below?

Reply:
Fix proposed in [`9472fa6` (#6575)](https://github.com/blakeblackshear/frigate/pull/6575/commits/9472fa691ea95855ef03fa631ae2736242a18203).